src/Infrastructure/LeanCode.IdentityServer.KeyVault/TokenCreationService.cs

```diff
@@ -12,10 +12,12 @@ internal sealed class TokenCreationService : DefaultTokenCreationService
 
     public TokenCreationService(
         SigningService signing,
+#pragma warning disable CS0618 // obsolete
```

Comment:
`SuppressMessage`? It's less disturbing than pragmas IMO.

Reply:
It'd be the whole constructor then, but yeah, it's obsolete anyway.